src/components/cards/VideoCard.jsx

```diff
@@ -4,11 +4,16 @@ import { MdWatchLater } from "react-icons/md";
 import { RiPlayList2Fill } from "react-icons/ri";
```

Comment:
Suggestion : You can have a file for importing all the icons then you can import them from there and use them .

Reply:
noted